src/compass/utils/schemas/geo_cslc_s1.yaml

```diff
@@ -0,0 +1,104 @@
+runconfig:
+    name: str()
+
+    groups:
+        pge_name_group:
+           pge_name: enum('CSLC_S1_PGE')
+
+        input_file_group:
+           # Required. List of SAFE files (min=1)
+           safe_file_path: list(str(), min=1)
+           # Required. List of orbit (EOF) files
+           orbit_file_path: list(str(), min=1)
+           # List of (unique) burst ID to process
+           burst_id: list(str(), min=1, required=True)
```

Comment:
@vbrancat my understanding is that geo_cslc.py processes only one burst (which can have one or more polarizations). Is this correct? If yes, shall we change list to str?

Reply:
That is correct. The current baseline is to process burst by burst. We can enforce the orbit file and the `burst_id` to be `str` instead of list of strings.

Reply:
I think it is ok to keep safe_file and orbit_file as "list". But burst_id should be "str" because the workflow handles one burst_id. 
The reason to keep afe_file and orbit_file as "list" would be the rare situation that multiple zip files are given and user is not sure which one has the bust_id.